docs/examples/Example_data_fusion_from_sensors.py

```diff
@@ -0,0 +1,506 @@
+EKF_updater = ExtendedKalmanUpdater(measurement_model=None)
+
+# define the hypothesiser
+hypothesiser_UKF = DistanceHypothesiser(
+    predictor=UKF_predictor,
+    updater=UKF_updater,
+    measure=Mahalanobis(),
+    missed_distance=20) # use a large distance measure
+
+# define the distance data associator
+data_associator_UKF = GNNWith2DAssignment(hypothesiser_UKF)
+
+# define the hypothesiser
+hypothesiser_EKF = DistanceHypothesiser(
+    predictor=EKF_predictor,
+    updater=EKF_updater,
+    measure=Mahalanobis(),
+    missed_distance=20) # use a large distance measure
+
+# define the distance data associator
+data_associator_EKF = GNNWith2DAssignment(hypothesiser_EKF)
+
+# define a track deleter based on time measurements
+deleter = UpdateTimeDeleter(timedelta(seconds=3), delete_last_pred=True)
+
+# create a track initiator placed on the target tracks origin
+UKF_initiator = MultiMeasurementInitiator(
+    prior_state=GaussianState([10, 0, 10, 0],
+                              np.diag([1, 1, 1, 1])),
+    measurement_model=None,
+    deleter=deleter,
+    updater=UKF_updater,
+    data_associator=data_associator_UKF)
+
+EKF_initiator = MultiMeasurementInitiator(
+    prior_state=GaussianState([10, 0, 10, 0],
+                              np.diag([1, 1, 1, 1])),
+    measurement_model=None,
+    deleter=deleter,
+    updater=EKF_updater,
+    data_associator=data_associator_EKF)
+
+# Instantiate the particle predictor, resampler and updater
+PF_predictor = ParticlePredictor(transition_model)
+resampler = ESSResampler(threshold=n_particles/2.)
+PF_updater = ParticleUpdater(measurement_model=None,
+                             resampler=resampler)
+
+hypothesiser_PF = DistanceHypothesiser(
+    predictor=PF_predictor,
+    updater=PF_updater,
+    measure=Mahalanobis(),
+    missed_distance=20)
+
+# define the data associator
+data_associator_PF = GNNWith2DAssignment(hypothesiser_PF)
+
+# To instantiate the track initiator we define a prior state
+# as gaussian state with the target track origin
+initiator_particles= SimpleMeasurementInitiator(
+    prior_state=GaussianState([10, 0, 10, 0],
+                 np.diag([5, 0.1, 5, 0.1]) ** 2),
+    measurement_model=None,
+    skip_non_reversible=True)
+
+# Particle filter initiator, use 1024 particles
+PF_initiator = GaussianParticleInitiator(
+    initiator=initiator_particles,
+    number_particles=n_particles)
+
+# Load the multitarget trackers
+from stonesoup.tracker.simple import MultiTargetTracker
+
+# Load a detection reader
+from stonesoup.buffered_generator import BufferedGenerator
+from stonesoup.reader.base import DetectionReader
+
+
+# Create a dummy detector to parse the detections
+class DummyDetector(DetectionReader):
+    def __init__(self, *args, **kwargs):
+        self.current = kwargs['current']
+
+    @BufferedGenerator.generator_method
+    def detections_gen(self):
+        yield self.current
+
+
+# Instantiate the Kalman Tracker, without
+# specifying the detector
```

Comment:
```suggestion
# Instantiate each of the Trackers, without
# specifying the detector
```

Reply:
This change is probably irrelevant if you reorganise by filter type as per above comment